AppKit/include/AppKit/NSSpeechSynthesizer.h

```diff
@@ -21,7 +21,21 @@
 #import <Foundation/NSObject.h>
 #import <Foundation/NSString.h>
 
-APPKIT_EXPORT NSString *const NSVoiceName;
+typedef NSString *NSVoiceGenderName;
+
+APPKIT_EXPORT NSVoiceGenderName const NSVoiceGenderFemale;
+APPKIT_EXPORT NSVoiceGenderName const NSVoiceGenderMale;
+
+typedef NSString *NSVoiceAttributeKey;
+
+APPKIT_EXPORT NSVoiceAttributeKey const NSVoiceName;
+APPKIT_EXPORT NSVoiceAttributeKey const NSVoiceGender;
+
```

Comment:
Don't forget to include `NSVoiceLocaleIdentifier`

```suggestion
APPKIT_EXPORT NSVoiceAttributeKey const NSVoiceLocaleIdentifier;

```